product_description_without_description_sale/models/product_template.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2024 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    use_description_sale = fields.Boolean(
+        "Use Sales Description",
+        help="If selected, Sales Description will be appended"
+        "to the sales order line description.",
+        default=lambda self: self._default_use_description_sale(),
+    )
+
+    @api.model
+    def _default_use_description_sale(self):
+        # Get the system parameter configuration
+        param = (
+            self.env["ir.config_parameter"]
+            .sudo()
+            .get_param(
+                "product_description_without_description_sale.default_use_description_sale"
+            )
+        )
+        if param == "0":
+            return False
+        return True
```

Comment:
Default is not false when installed

Reply:
You need to load without demo data. If not, demo data will replace standard one.

Reply:
I try to install no demo db but it become true.  And make sure that system param is 0